eng/pipelines/insert.yml

```diff
@@ -203,7 +206,8 @@ steps:
         -updateAssemblyVersions "(default)" `
         -updateCoreXTLibraries "(default)" `
         -visualStudioBranchName "$(Template.VSBranchName)" `
-        -writePullRequest "prid.txt"
+        -writePullRequest "prid.txt" `
+        -queueSpeedometerValidation "${{ parameters.queueSpeedometerValidation }}"
```

Comment:
A dumb question: 
Why it's `${{ parameters.queueSpeedometerValidation }}` instead of `$(parameters.queueSpeedometerValidation)`

Reply:
It is an azure devops template parameter and not a powershell script parameter.  ${{ xyz }} is the syntax for azure devops template parameters